x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -139,6 +146,12 @@ export class HeadlessChromiumDriverFactory {
           ignoreHTTPSErrors: true,
           handleSIGHUP: false,
           args: chromiumArgs,
+
+          // We optionally set this at page creation to reduce the chances of
+          // browser reflow. In most cases only the height needs to be adjusted
+          // before taking a screenshot.
+          // NOTE: _.defaults assigns to the target object, so we copy it.
+          defaultViewport: _.defaults({ ...defaultViewport }, DEFAULT_VIEWPORT),
```

Comment:
Is there a reason why this can't be just:
```suggestion
          defaultViewport: { ...DEFAULT_VIEWPORT, ...defaultViewport },
```

Reply:
The reason I chose `_.defaults` is because the override behaviour is different with own props (it ignores overrides that are `undefined`, so we won't end up with `width: undefined` for ex.).

Do you think we should change the type of `defaultViewport` making both `width` and `height` required values? I see they are both required in the `Size` object. WDYT?